examples/pom.xml

```diff
@@ -22,7 +22,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.shardingsphere.example</groupId>
     <artifactId>shardingsphere-example</artifactId>
-    <version>5.0.1-SNAPSHOT</version>
+    <version>${revision}</version>
```

Comment:
Hi @Swastyy 
Sorry for the delay. You may need to define a value for `revision` in properties.

Reply:
Yea, I have added that 👍